src/rleCompress.js

```diff
@@ -18,7 +18,20 @@
  * @return {string}
  */
 function rleCompress(source) {
-  // write code here
+  let count = 1;
+  let compress = '';
+
+  for (let i = 0; i < source.length; i++) {
+    if (source[i] === source[i + 1]) {
+      count++;
+    } else if (source[i] !== source[i + 1] && count > 1) {
```

Comment:
ця перевірка не має сенсу якщо перший if не відпрацював
```suggestion
    } else if ( count > 1) {
```